spec/API_specification/indexing.md

```diff
@@ -114,16 +114,18 @@ Using a slice to index a single array axis must adhere to the following rules. L
 
 ```{note}
 
-This specification does not require "clipping" out-of-bounds indices (i.e., requiring the starting and stopping indices `i` and `j` be bound by `0` and `n`, respectively).
+This specification does not require "clipping" out-of-bounds slice indices (for example, in Python slice semantics, `0:100` and `0:10` are equivalent on a list of length `10`).
```

Comment:
```suggestion
This specification does not require "clipping" out-of-bounds slice indices.

```{note}

This is in contrast to Python slice semantics where `0:100` and `0:10` are equivalent on a list of length `10`.
```
```